src/LodeRunner.API/src/Handlers/ResultHandler.cs

```diff
@@ -188,7 +304,7 @@ public static async Task<JsonResult> CreateErrorResult(string message, HttpStatu
         /// <param name="statusCode">The http code.</param>
         /// <param name="contentType">Json Content Type.</param>
         /// <returns>the Json Result.</returns>
-        public static async Task<JsonResult> CreateResult<TEntity>(TEntity data, HttpStatusCode statusCode, string contentType = JsonContentTypeApplicationJson)
+        public static async Task<JsonResult> CreateResult<TEntity>(TEntity data, HttpStatusCode statusCode, string contentType = "application/json")
```

Comment:
revert back?
```suggestion
        public static async Task<JsonResult> CreateResult<TEntity>(TEntity data, HttpStatusCode statusCode, string contentType = JsonContentTypeApplicationJson)
```

Reply:
I switched it out just because this function will be deleted with the last method (delete) refactor, and I didn't want to accidentally leave an unused variable. Thoughts on keeping it for this PR, since the next one should delete this function?